### PR TITLE
[PLATFORM-1443] Create an FAQ

### PR DESCRIPTION
**Pull request by chandlma**
Created an FAQ per https://frame-io.atlassian.net/browse/PLATFORM-1443

## Types of changes

This is a new file.

## Checklist: Should we create a new one for docs? 

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/guides/FAQ/FAQ -11-24
Comment (jwixn) [inline src/pages/guides/FAQ/FAQ -11-24:3]:
```suggestion
The Frame V4 API allows developers to integrate Frame.io’s surface for creatives and their teams to manage their work for every step of the asset delivery lifecycle from plan to delivery.
```
Personally, I don't think we need to try to summarize what the API can be used for in this document, but I'll defer to marketing and product folks for their best judgement on that front 😄

File: src/pages/guides/FAQ/FAQ -11-24
Comment (jwixn) [inline src/pages/guides/FAQ/FAQ -11-24:15]:
```suggestion
    * Reach out to the Frame.io support team for assistance by chatting with us on our [Support Site](https://help.frame.io/en/). Please note that enterprise customers may be directed to their Customer Success Manager.
```
I guess Quip helpfully converted these into links for you when you exported as markdown, but we should probably fix all of these before merging / publishing.

File: src/pages/guides/FAQ/FAQ -11-24
Comment (jwixn) [inline src/pages/guides/FAQ/FAQ -11-24:11]: I'm not sure we need to describe this process again here. My personal preference if we include this question would be just to link to the description in the Getting Started guide (or elsewhere). My rationale is that, just as with code, it's harder to keep multiple descriptions up-to-date as we make changes in the future.

File: src/pages/guides/FAQ/FAQ -11-24
Comment (jwixn) [inline src/pages/guides/FAQ/FAQ -11-24:23]:
```suggestion
    * **Open API Standards**: A REST API and an OpenAPI specification connect Frame.io to the systems you already use to unblock custom workflows.
```
We don't support any native integrations yet in our V4 API (at least as part of an officially supported SDK) so I think we may be getting ahead of ourselves here 😉

File: src/pages/guides/FAQ/FAQ -11-24
Comment (jwixn) [inline src/pages/guides/FAQ/FAQ -11-24:27]: Are we just saying that clients should use the Frame Desktop Transfer app here? This doesn't apply directly to the files that reside in Azure Blob storage though. At the moment, the V4 API doesn't support transferring folders or files in bulk.

I think we (dev platform squad) may need to craft a better (or more thorough) recommendation for you to describe here.

File: src/pages/guides/FAQ/FAQ -11-24
Comment (jwixn) [inline src/pages/guides/FAQ/FAQ -11-24:31]: Not sure what Frame marketing thinks on this subject, but I think we should capitalize Metadata consistently when used in the context or maybe even go further and say "Frame.io Metadata". 

Additionally, at the moment we are referring to  Frame Metadata  as Custom Fields in our V4 API, but I think there has been some talk of changing this, so we probably need to nail this down as well.

File: src/pages/guides/FAQ/FAQ -11-24
Comment (jwixn) [inline src/pages/guides/FAQ/FAQ -11-24:37]: I'm wondering if we're beyond the scope here of what I would expect would be frequently asked questions about the V4 API.

File: src/pages/guides/FAQ/FAQ -11-24
Comment (jwixn) [inline src/pages/guides/FAQ/FAQ -11-24:41]: I think this is made redundant with by the sentence that precedes it.

File: src/pages/guides/FAQ/FAQ -11-24
Comment (jwixn) [inline src/pages/guides/FAQ/FAQ -11-24:48]: I think some bullets got lumped together here.

File: src/pages/guides/FAQ/FAQ -11-24
Comment (jwixn) [inline src/pages/guides/FAQ/FAQ -11-24:53]: I think we need a clearer answer before we publish this 😄

File: src/pages/guides/FAQ/FAQ -11-24
Comment (jwixn) [inline src/pages/guides/FAQ/FAQ -11-24:55]: I think we should say it's in development if we expect that we'll be updating these integrations in the relatively near future. Otherwise we should be clear that this is something that we expect the community to contribute.

**Review by jwixn — COMMENTED** [review]
I left a bunch of thoughts, but all are intended to be constructive and only reflect my fuzzy, non-expert opinions when it comes to prose and what belongs in this document 😄 Thanks for making progress on this!

**Comment by chandlma** [comment]
Updated with Julian's comments.

File: src/pages/guides/FAQ/FAQ -11-24
Comment (chandlma) [reply src/pages/guides/FAQ/FAQ -11-24:3]: I took this from marketing and hear you so I'll just shorten it up.

File: src/pages/guides/FAQ/FAQ -11-24
Comment (chandlma) [reply src/pages/guides/FAQ/FAQ -11-24]: Done!

File: src/pages/guides/FAQ/FAQ -11-24
Comment (chandlma) [reply src/pages/guides/FAQ/FAQ -11-24:10]: I took it out.

File: src/pages/guides/FAQ/FAQ -11-24
Comment (chandlma) [reply src/pages/guides/FAQ/FAQ -11-24:26]: I'll take it our for now.

File: src/pages/guides/FAQ/FAQ -11-24
Comment (chandlma) [reply src/pages/guides/FAQ/FAQ -11-24:47]: Fixed.

File: src/pages/guides/FAQ/FAQ -11-24
Comment (chandlma) [reply src/pages/guides/FAQ/FAQ -11-24:40]: Took it out.

File: src/pages/guides/FAQ/FAQ -11-24
Comment (chandlma) [reply src/pages/guides/FAQ/FAQ -11-24:36]: Took out non-V4 API topics.

File: src/pages/guides/FAQ/FAQ -11-24
Comment (chandlma) [reply src/pages/guides/FAQ/FAQ -11-24:30]: Took out Metadata for now.

**Comment by tmartin8080** [comment]
Can we rename this file to `index.md` inside the FAQ directory? 

Screenshot 2024-12-02 at 2 16 37 PM

File: src/pages/guides/FAQ/FAQ -11-24
Comment (jhodges10) [inline src/pages/guides/FAQ/FAQ -11-24:12]: We should investigate whether this can be done as a relative link or injected using environment variables so it will work both in the staging and production environments.

Comment (chandlma) [reply src/pages/guides/FAQ/FAQ -11-24:6]: Thanks Jeff. I'll look to you to lead that effort and report back!

File: src/pages/guides/FAQ/FAQ -11-24
Comment (chandlma) [reply src/pages/guides/FAQ/FAQ -11-24]: Done!

**Review by tmartin8080 — APPROVED** [review]

**Pull request merged**
